### ISSUE TITLE
The layout of Data table header test is bad for translation language (zh-Hant, zh-Hans, ko)

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#5817]
## Title line template: [Title]: Brief description

The layout of Data table header text is bad for translation language

## Environment

> Operating system
 Mac OS Catalina 10.15.3 (19D76)

> Browser
Version 80.0.3987.87

## Detailed description

> What version of the Carbon Design System are you using?
"@console/pal": "1.112.1"

> What did you expect to happen?
The translation string should be display in single line, not word wrap by each characters.

> What happened instead?
The translated string display in vertical line

## Steps to reproduce the issue
1.Step one
Set browser language to zh-Hant or zh-Hans or ko and open datatable component

2.Step two
Resize browser or zoom out below 80%

## Additional information

- Screenshots or code
zh-Hant
![image](https://user-images.githubusercontent.com/46310374/78627145-5d1c4d80-78ae-11ea-8232-fa45bf9be5b4.png)

zh-Hans
![image](https://user-images.githubusercontent.com/46310374/78627156-63aac500-78ae-11ea-8add-62f6c638d626.png)

ko
![image](https://user-images.githubusercontent.com/46310374/78627173-6efdf080-78ae-11ea-9627-57423eeb7937.png)

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by reeteshsingh93** [comment]
Code sanbox: https://codesandbox.io/s/dank-mountain-wysox?file=/src/index.js
Try open the page in new tab and resize the browser to see the issue:
![image](https://user-images.githubusercontent.com/46310374/79189237-d8e24100-7e3e-11ea-8dbe-dfe118ee73dc.png)

**Comment by joshblack** [comment]
Looking into this, it seems like this is browser behavior related to overflow/resizing. What would the preference be for how this should render? 👀 It seems like if they render on the same line then the container would overflow in the x direction 🤔

**Comment by reeteshsingh93** [comment]
For other languages it comes in another line.

**Comment by emyarod** [comment]
> For other languages it comes in another line.

do you mean like this?

![image](https://user-images.githubusercontent.com/8265238/79353522-c3cde680-7f00-11ea-9b49-37ff83d5e203.png)

**Comment by reeteshsingh93** [comment]
Yes it should come like the above snapshot in all the resolution as per our accessibility testers.

**Comment by emyarod** [comment]
this is the default behavior depending on the width of the table header and container elements. [this](https://en.wikipedia.org/wiki/Line_breaking_rules_in_East_Asian_languages) may also be a useful reference regrading line breaks with CJK characters
